Automattic/simplenote-android
# Update Empty Views

### Fix
Update the empty views throughout the app by adding images and changing the copy.  See the screenshots below for illustration.

![update_empty_views_light](https://user-images.githubusercontent.com/3827611/68521181-b890d380-025b-11ea-9a94-896dc136147c.png)

![update_empty_views_dark](https://user-images.githubusercontent.com/3827611/68521184-ba5a9700-025b-11ea-9e30-36539f32b420.png)

#### Note
These changes do not include the updated gray colors used for ***Dark***.  Those are addressed in a separate pull request.

### Test
1. Move all notes to ***Trash***.
2. Tap ***All Notes*** in navigation drawer.
3. Notice ***Create your first note*** empty view.
4. Tap ***Untagged Notes*** in navigation drawer.
5. Notice ***Create an untagged note*** empty view.
6. Tap any tag under ***Tags*** section in navigation drawer.
7. Notice ***No notes tagged "TAG"*** empty view where TAG is tag selected in Step 4.
8. Restore all notes from ***Trash***.
9. Tap ***Trash*** in navigation drawer.
10. Notice ***Your trash is empty*** empty view.
11. Tap ***All Notes*** in navigation drawer.
12. Tap ***Search*** action in top app bar.
13. Enter any text not matching notes in search field.
14. Submit search with entered text.
15. Notice ***No notes found*** empty view.
16. Tap back arrow in top app bar.
17. Tap ***Edit*** button under ***Tags*** section in navigation drawer.
18. Tap delete button next to all tags.
19. Notice ***No tags*** empty view.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
These changes do not require release notes.
